package.json

```diff
@@ -11,44 +11,43 @@
   "license": "Apache-2.0",
   "scripts": {
     "prepublish": "typings install && npm run prod",
-    "lint": "tslint -c tslint.json src/*.ts src/*.tsx src/**/*.ts src/**/*.tsx src/**/**/*.ts src/**/**/*.tsx && sass-lint -c .sass-lint.yml -v -q",
-    "test": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js lib/__tests__/*.js lib/**/__tests__/*.js lib/**/**/__tests__/*.js",
+    "lint": "tslint -c tslint.json src/*.ts src/*.tsx src/components/*.ts src/components/*.tsx && sass-lint -c .sass-lint.yml -v -q",
```

Comment:
I think the `src/**/*.ts` is more flexible in case we added another directory next to `components`. However, if this repo is all components maybe we should flatten it out and put everything directly under `src`.

Reply:
Good point. I'll leave that for Adriana to decide since she's working on the components for this repo.

Reply:
I do anticipate the repo being all components, but I like having components and stylesheets in separate folders to make it easier to find things.